Test/MetaDrawTest.cs

```diff
@@ -54,5 +55,40 @@ public static void TestMetaDrawReadCrossPsmFile()
 
             Directory.Delete(folderPath, true);
         }
+
+        [Test]
+        public static void TestLabelingVariantCrossingIons()
+        {
+            string myFile = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestData\VariantCrossTest.psmtsv");
+            List<string> warnings = new List<string>();
+            List<PsmFromTsv> psms;
+
+            psms = PsmTsvReader.ReadTsv(myFile, out warnings);  //test will fail if order of psms isn't from top to bottom row
```

Comment:
Check that you are parsing everything in correctly now that you have added the identified variant field. Add assert statements to ensure that these are as expected.

Reply:
I added checks for Intersecting Sequence Variations, Identified Sequence Variations, and Splice Sites.